astromon/web/celmon.py

```diff
@@ -20,7 +20,7 @@
 from cxotime import CxoTime, units as u
 
 plt.rcParams['font.size'] = '16'
-
+plt.rcParams['figure.max_open_warning'] = 100
```

Comment:
Not sure of the exact context here and if it might help, but I found that closing each figure after writing via `fig.close()` is an effective way to prevent this warning.